src/components/TodoInfo/TodoInfo.jsx

```diff
@@ -1,4 +1,21 @@
-// Add the required props
-export const TodoInfo = () => (
-  <>TodoInfo markup</>
-);
+import './Todoinfo.scss';
+import cn from 'classnames';
+import { UserInfo } from '../UserInfo';
+
+export const TodoInfo = ({ todo }) => {
+  const {
+    title,
+    completed,
+    user,
+  } = todo;
+
+  return (
+    <article
+      className={cn('TodoInfo', { 'TodoInfo--completed': completed })}
+    >
+      <h2 className="TodoInfo__title">{title}</h2>
+
+      <UserInfo user={user} />
```

Comment:
`UserInfo` takes a `user` and renders the details (**use it only if todo.user exists**).

Reply:
Yurii mean to do like this: `user && <UserInfo user={user} />`